spot_driver/spot_driver/spot_ros2.py

```diff
@@ -2343,7 +2349,7 @@ def populate_camera_static_transforms(self, image_data: image_pb2.Image) -> None
         # We exclude the odometry frames from static transforms since they are not static. We can ignore the body
         # frame because it is a child of odom or vision depending on the preferred_odom_frame, and will be published
         # by the non-static transform publishing that is done by the state callback
-        frame_prefix = MOCK_HOSTNAME + "/"
+        frame_prefix = ""
```

Comment:
I think we need to duplicate this: https://github.com/bdaiinstitute/spot_wrapper/blob/ae0f808eeb9c2792dd44e83fc5bc8863f2f687b3/spot_wrapper/wrapper.py#L422